adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -396,17 +415,34 @@ case class RDDBoundFeatureRDD private[rdd] (
     sqlContext.createDataset(rdd.map(FeatureProduct.fromAvro))
   }
 
+  /**
+   * Replaces the sequence dictionary attached to a RDDBoundFeatureRDD.
+   *
+   * @param newSequences The new sequence dictionary to attach.
+   * @return Returns a new RDDBoundFeatureRDD with the sequences replaced.
+   */
   def replaceSequences(newSequences: SequenceDictionary): FeatureRDD = {
     copy(sequences = newSequences)
   }
 
+  /**
+   * Replaces the sample metadata attached to the RDDBoundFeatureRDD.
+   *
+   * @param newSamples The new sample metadata to attach.
+   * @return A RDDBoundFeatureRDD with new sample metadata.
+   */
+  def replaceSamples(newSamples: Iterable[Sample]): FeatureRDD = {
```

Comment:
This method is defined with scaladoc in `MultisampleGenomicDataset`, could we use `override def` instead?